### DISCUSSION TITLE
如何退出skynet

### DISCUSSION THREAD

**Discussion by leonlyd** [cloudwu/skynet#1788] [General]
skynet好像是在所有服务中止后都会退出
我在第一个main服务里只有一句话
skynet.exit();
但是在系统中skynet这个进程依然运行
每次运行编写代码我都需要kill进程
如何在skynet中已经没有运行任何自定义服务的情况下，让skynet关闭进程

**Comment by cloudwu** [comment]
https://github.com/cloudwu/skynet/wiki/APIList
